pom.xml

```diff
@@ -183,6 +183,10 @@
         <dep.objenesis.version>1.3</dep.objenesis.version>
         <dep.slice.version>0.10</dep.slice.version>
         <dep.jmh.version>1.15</dep.jmh.version>
+        <dep.junit.jupiter.version>5.0.0-RC2</dep.junit.jupiter.version>
```

Comment:
I think our convention is to name these `dep.NAME.version` so let's call this `dep.junit-jupiter.version`. That will match with `assertj-guava`, etc.